blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,183 @@
+          link: url,
+          guid: s._id,
+          description: {
+            body: {
+              'content:encoded':
+                {
+                  $: body,
+                } || '',
+            },
+          },
+          pubDate: moment
+            .utc(s[pubDate])
+            .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+          ...(itemCategory &&
+            (category = jmespath.search(s, itemCategory)) &&
+            category && { category: category }),
+          ...(includeContent !== 0 &&
+            (body = rssBuildContent.parse(
+              s.content_elements || [],
+              includeContent,
+              domain,
+              resizerKey,
+              resizerURL,
+              resizerWidth,
+              resizerHeight,
+            ))),
+          ...(includePromo &&
+            img &&
+            img.url && {
+              'media:content': {
+                '@type': 'image/jpeg',
+                '@url': buildResizerURL(
+                  img.url,
+                  resizerKey,
+                  resizerURL,
+                  resizerWidth,
+                  resizerHeight,
+                ),
+                ...(jmespath.search(img, imageCaption) && {
+                  'media:description': {
+                    '@type': 'plain',
+                    $: jmespath.search(img, imageCaption),
+                  },
+                }),
+                ...(jmespath.search(img, imageTitle) && {
+                  'media:title': {
+                    $: jmespath.search(img, imageTitle),
+                  },
+                }),
+                ...((jmespath.search(img, imageCredits) || []).length && {
+                  'media:credit': {
+                    '@role': 'author',
+                    '@scheme': 'urn:ebu',
+                    '#': jmespath.search(img, imageCredits).join(','),
+                  },
+                }),
+              },
+            }),
+        }
+      }),
+    },
+  },
+})
+
+export function Rss({ globalContent, customFields, arcSite }) {
+  const {
+    resizerURL = '',
+    feedDomainURL = '',
+    feedTitle = '',
+    feedLanguage = '',
+  } = getProperties(arcSite)
+  const { width = 0, height = 0 } = customFields.resizerKVP || {}
+
+  const rssBuildContent = new BuildContent()
+
+  // can't return null for xml return type, must return valid xml template
+  return rssTemplate(get(globalContent, 'content_elements', []), {
+    ...customFields,
+    resizerURL,
+    resizerWidth: width,
+    resizerHeight: height,
+    domain: feedDomainURL,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  })
+}
+
+Rss.propTypes = {
+  customFields: PropTypes.shape({
+    channelPath: PropTypes.string.tag({
+      label: 'Path',
+      group: 'Channel',
+      description:
+        'Path to the feed, excluding the domain, defaults to /arc/outboundfeeds/rss',
+      defaultValue: '/arc/outboundfeeds/rss/',
+    }),
+    audioAvailable: PropTypes.kvp.tag({
```

Comment:
We can focus on this once the rest of the feed is working

Reply:
This should be a string
Label: Enter jmespath for the ANS element to include in enclosure tag
defaultValue: content_elements[?type==audio].streams[0].url'